snapcraft.yaml

```diff
@@ -0,0 +1,21 @@
+name: mkcert
+version: '1.0'
```

Comment:
This should match the latest released version, right? That's `1.1.2` as of today.

Reply:
I used 1.0 as a first build tag, you can change it to whatever is most appropriate. In snapcraft, version does not carry a semantic meaning. So you can match it to your application version, git branch, etc.